common/app/targeting/TargetingLifecycle.scala

```diff
@@ -0,0 +1,29 @@
+package targeting
+
+import app.LifecycleComponent
+import common.{JobScheduler, AkkaAsync}
+import play.api.inject.ApplicationLifecycle
+
+import scala.concurrent.{Future, ExecutionContext}
+
+class TargetingLifecycle(
+  appLifecycle: ApplicationLifecycle,
+  jobs: JobScheduler,
+  akkaAsync: AkkaAsync)(implicit executionContext: ExecutionContext) extends LifecycleComponent {
+
+  appLifecycle.addStopHook { () => Future {
+    jobs.deschedule("TargetingCampaignRefreshJob")
+  }}
+
+  override def start(): Unit = {
+    jobs.deschedule("TargetingCampaignRefreshJob")
+    jobs.schedule("TargetingCampaignRefreshJob", "0 * * * * ?") {
```

Comment:
better not to use schedule, as its is extremely difficult to check it's sensible. There is a scheduleEvery method that takes a duration so you can pass in 1.hour or something and everyone can see what it means by inspection